code/style.css

```diff
@@ -143,7 +146,65 @@ button {
   transition: all 0.3s ease;
 }
 
+#colaBtn {
+  background-color: #B81D13;
+  color: black;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+#fantaBtn {
+  background-color: #EFB700;
+  color: black;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+#spriteBtn {
+  background-color: #008450;
+  color: black;
+  font-size: 20px;
+  font-weight: bold;
+}
+
 button:hover {
   opacity: 0.9;
   transition: all 0.2s ease;
 }
+
+select {
+  border: none;
+  border-radius: 4px;
+  color: #AA5656;
+  font-size: 16px;
+  font-weight: 600;
+  height: 60px;
+  font-family: 'Montserrat';
+  background: #e5e9ff;
+  width: 100%;
+  padding: 0 20px;
+  cursor: pointer;
+  text-align: center;
+}
+
+footer {
+  color: white;
+  font-size: 5px;
+  margin: 50px;
+  text-align: center;
+}
+
+footer a {
+  text-decoration: none;
+  color: #F1DBBF;
+}
+
+footer a:hover {
+  text-decoration: underline;
+}
+
+@media only screen and (max-width: 600px) {
+  h1 {
+    font-size: 35px;
+  }
+}
```

Comment:
I did not see a requirement of having this project responsive, but it would be an easy fix to give the input field a display:flex, flew-wrap or flex-direction:column property when on a mobile device. Or make the buttons have less padding when on a mobile device. Right now the buttons and the input field stretches outside of the container for around 400px. The other parts of the page is responsive and looks nice in all sizes :)